src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -225,19 +157,16 @@ frmelmnts.lbl.anncmnttblseen = Seen
 frmelmnts.lbl.anncmnttblsent = Sent
 frmelmnts.lbl.attributions = Attributions
 frmelmnts.lbl.author = Author
-frmelmnts.lbl.authorofsourcecontent = Author of Source Content
```

Comment:
This string is in use. : frmelmnts.lbl.authorofsourcecontent = Author of Source Content
content-credits.component.html